config/routes.rb

```diff
@@ -3,4 +3,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :users,:posts,:likes,:comments
```

Comment:
- [ ] I suggest that you should make sure that your application handles only the 4 required URLs. :point_down: as requested in the project requirement.

>  Make sure that when you run the rails routes command in your console, you can see the following output
  ![image](https://github.com/David-Lanzz/lanzz-blog/assets/92303891/5b47a454-ee8b-4a2b-a6dc-f3ff356ca5ee)